V2 docs

isaacabraham opened the pull request:
Basic docs for the new template:

* Introduction
* Getting Started
* Template

isaacabraham commented:
@theimowski Note that we have a large branch with a number of recipes ready-to-go once this goes in (see [here](https://github.com/CompositionalIT/safe-stack-docs)).

theimowski commented:
wondering if we should maybe leave current content under a "v1 docs" tab for some time - there might be users still wanting to have docs for version 1?

isaacabraham commented:
@theimowski hmmmm. That's a good idea - how should we do that though? Could we host two versions of the docs?

Think about it though, the majority of the docs were mostly around the template options etc. - the new docs will still contain the stuff about how SAFE works etc., it's just going to be in a different shape.

isaacabraham commented:
@theimowski Just so I'm clear - are you waiting for me to do this? I'm not sold on duplicating / keeping the old docs public - I'm happy to put a link to the previous github commit for people to look through the markdown if needed, but otherwise tbh I'd like to release the template and docs ASAP :-)

theimowski commented on mkdocs.yml:
```diff
@@ -35,17 +35,7 @@ nav:
   - Learn about Elmish : 'component-elmish.md'
 - The SAFE Template:
   - Overview: 'template-overview.md'
-  - Run and Build the app: 'template-fake.md'
```
How about just the following:
* Leave the files that have been removed
* Prefix them with "legacy-*" so that they can be easily identified
* Create new section at the very end of this list "Legacy SAFE Template (v1)" and put all these pages there?

theimowski commented:
Added a comment how we can do this with minimal effort - if you don't want to do that, I'm happy to do it on top of your PR then :)

isaacabraham commented:
If you can, that would be great :-)

theimowski commented on mkdocs.yml:
```diff
@@ -60,7 +50,6 @@ nav:
   - Working with Azure functions : 'feature-azurefunctions.md'
 - FAQs:
   - Choosing your web server : 'faq-server.md'
-  - Moving from dev to prod : 'faq-build.md'
```
why this has been removed? Quickly browsing through that page it seems to be applicable to v2 as well, or am I missing something?

theimowski commented:
https://github.com/SAFE-Stack/docs/pull/161

isaacabraham commented:
There's a ready-to-merge in recipe which is much better at showing how to move from dev to prod. I was going to also rewrite the removed article as more of a reference guide. You can see all of them [here](https://github.com/CompositionalIT/safe-stack-docs) in the recipes branch.

isaacabraham commented:
Fixes #161

isaacabraham commented:
@theimowski I've fixed the build - good old tabs instead of spaces YML fun :-) And also added a missing file (google cloud registration). There's still more that we'll need to add over time but I believe with this branch and the recipes we have ready to go (over 30) that we are in a good place.

isaacabraham commented:
Shall I leave this for you to merge in when you've released the nuget package?

theimowski commented:
yup - in case of docs it's much easier - should be rolled out automatically after merging
